test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/GetInteriorVehicleData/013_Absence_of_isSubscribed_parameter_in_case_request_from_Mob_app_without_subscribe_parameter_but_response_from_HMI_with_isSubscribed.lua

```diff
@@ -0,0 +1,80 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/smartdevicelink/sdl_requirements/issues/2
+-- Use case: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/current_module_status_data.md
+-- Item: Use Case 1: Main Flow
+--
+-- Requirement summary:
+-- [SDL_RC] Current module status data GetInteriorVehicleData
+--
+-- Description: TRS: GetInteriorVehicleData, #2
+-- In case:
+-- 1) RC app sends valid and allowed-by-policies GetInteriorVehicleData_request without "subscribe" parameter
+-- 2) and SDL gets GetInteriorVehicleData_response with resultCode: <"any-result">
+-- 3) and with "isSubscribed" parameter from HMI
+-- SDL must:
+-- 1) transfer GetInteriorVehicleData_response with resultCode: <"any-result">
+-- and without "isSubscribed" param to the related app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/commonRCmodules')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+--[[ Local Functions ]]
+local function getDataForModule(pModuleType, isSubscriptionActive)
+  local mobileSession = common.getMobileSession()
+  local cid = mobileSession:SendRPC("GetInteriorVehicleData", {
+      moduleType = pModuleType
+      -- no subscribe parameter
+    })
+
+  EXPECT_HMICALL("RC.GetInteriorVehicleData", {
+      appID = common.getHMIAppId(),
+      moduleType = pModuleType
+    })
+  :Do(function(_, data)
+      common.getHMIconnection():SendResponse(data.id, data.method, "SUCCESS", {
+          moduleData = common.getModuleControlData(pModuleType),
+          isSubscribed = isSubscriptionActive -- return current value of subscription
+        })
+    end)
+  :ValidIf(function(_, data) -- no subscribe parameter
+      if data.params.subscribe == nil then
+        return true
+      end
+      return false, 'Parameter "subscribe" is transfered with to HMI value: ' .. tostring(data.params.subscribe)
+    end)
+
+  mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS",
+      moduleData = common.getModuleControlData(pModuleType)
+    })
+  :ValidIf(function(_, data) -- no isSubscribed parameter
```

Comment:
@dboltovskyi @GetmanetsIrina in case when app is already subscribed to module type, and we send GetInteriorVehicleData without subscribe parameter, this parameter will we false (default value from API),
so we must transfer isSubscribed param from HMI response to app
https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/TRS/RC/GetInteriorVehicleData.md
(Use case 8, 9)

Reply:
so I think that this script should be updated to something like 
https://github.com/LitvinenkoIra/sdl_atf_test_scripts/blob/feature/new_remote_control_modules/test_scripts/RC/AUDIO_LIGHT_HMI_SETTINGS/GetInteriorVehicleData/013_Absence_of_isSubscribed_parameter_in_case_request_from_Mob_app_without_subscribe_parameter_but_response_from_HMI_with_isSubscribed.lua